### DISCUSSION TITLE
Bitrate/quality parameter

### DISCUSSION THREAD

**Discussion by l3iggs** [ChristopherKlay/StadiaEnhanced#159] [Ideas & Requests]
Would it be possible to present the bitrate or quality parameter the stream was encoded with in the stream info section?

**Comment by ChristopherKlay** [comment]
Those infos are sadly not available via the WebRTC stream.

The only source of data we have is the connection used itself and the video stream. The bitrate of the current video stream is indirectly included in the bandwidth usage, but it's not a good indicator of quality.

**Comment by l3iggs** [comment]
That is sad! I'm surprised it's not metadata included in the stream itself. I will start looking for it!

**Comment by l3iggs** [comment]
I wonder if there's some good info here in `qpSum` (which I guess you should be able to see).

```
	// QPSum is the sum of the QP values of frames passed. The count of frames is
	// in FramesDecoded for inbound stream stats, and in FramesEncoded for outbound stream stats.
```
QP or Quantization Parameter, might be a pretty good indicator of quality, with lower being better.

But actually (looking at the code) looks like you've already got this exact thing in there with the "Compression" metric. So maybe that's the best thing to look at if I want to somehow quantify the stream's quality.
